docs/examples/kube2iam.rst

```diff
@@ -0,0 +1,242 @@
+Kube2IAM
+--------
+
+Kube2IAM is an extension to kubernetes that will allow you to give
+fine grained AWS access to pods without. More information about the
+project can be found on the `project page <https://github.com/jtblin/kube2iam>`_.
+
+Prerequisite
+~~~~~~~~~~~~
+
+Make sure `HELM <https://www.helm.sh/>`_ is `activated <https://docs.tarmak.io/user-guide.html#tiller>` on the Tarmak cluster.
+You also need to make sure you can connect to the cluster with your HELM install.
+
+.. code-block:: bash
+
+    helm version
```

Comment:
how can I connect to tarmak using helm? The process of retrieving a kubeconfig is not clear here - I had to hack the kubeconfig generated by `tarmak kubectl get nodes` and exposed the LB publically

Reply:
that will be part of #279. I will make sure to add an reference in these docs once that will be done.